.travis.yml

```diff
@@ -27,21 +27,60 @@ notifications:
 
 jobs:
   include:
+    - name: "Documentation Update"
+      script: 'if [ "$TRAVIS_PULL_REQUEST" = "false" ]; then ./.travis/travis-docs.sh; fi'
+      arch: amd64
+
+
+    - name: "CentOS 7"
+      script: ./.travis/travis-centos.sh
+      arch: amd64
+
+
     - name: "Fedora 29"
```

Comment:
One more suggestion... When looking at the travis results summary, it would be good to know which builds were x86_64 vs. aarch64. You should be able to add the platform to the Fedora job names (eg., "Fedora 29 x86_64" here and ""Fedora 29 aarch64" below). The `travis-fedora.sh` script already does the right thing by setting the release to `${arr[1]}` and ignoring other tokens in the job name.

Reply:
Interesting; the Travis view when you aren't signed in doesn't show the architecture field in the build overview. It shows up for me while I'm signed in.

Reply:
Yes, interesting. With the build stages you just added (:+1:, btw), the architecture field now shows up without being signed in--in addition to appearing in the job name.